### ISSUE TITLE
plugin 121 breaks subsequent plugins

### ISSUE THREAD

**Issue by fabiofreireny** [alanrenouf/vCheck-vSphere#662]
Plugins that are executed after plugin 121 don't produce expected results, or throw error messages

**Comment by danbarr** [comment]
Correct. @fabiofreireny submitted PR #647 and I suggested opening an issue to link and get more visibility for it.
